src/modules/75-ce/components/Connectors/AWSCOConnector/steps/OverviewStep.tsx

```diff
@@ -142,7 +142,7 @@ const OverviewStep: React.FC<StepProps<ConnectorConfigDTO> & OverviewStepProps>
           })
         })}
       >
-        {formikProps => {
+        {(formikProps: any) => {
```

Comment:
Codacy has a fix for the issue: [Unexpected any. Specify a different type.](https://app.codacy.com/gh/harness/harness-core-ui/pullRequest?prid=9232251)
```suggestion
        {(formikProps: unknown) => {
```